### PR TITLE
Special Summary Links

### PR DESCRIPTION
**Pull request by sbddesign**

### CODE REVIEW

File: components/MDX-components.tsx [hunk 1]
```diff
@@ -6,163 +6,187 @@ import { cn } from "@/lib/utils"
...
+    h6: ({ className, ...props }: { className?: any}) => (
+      <h6
+        className={cn(
+          "mt-8 scroll-m-20 text-base font-semibold tracking-tight",
+          className
+        )}
+        {...props}
+      />
+    ),
+    // a: ({ className, ...props }: { className?: any}) => (
```
Comment (jordan-bravo) [inline components/MDX-components.tsx:95]: Is this commented code wanted?

File: app/posts/[slug]/page.tsx [hunk 2]
```diff
@@ -27,9 +27,11 @@ const page = async ({ params }: PageProps) => {
     
     console.log(post._raw.flattenedPath)
```
Comment (jordan-bravo) [inline app/posts/[slug]/page.tsx:28]: Console log wanted?

File: app/posts/[slug]/page.tsx [hunk 3]
```diff
@@ -27,9 +27,11 @@ const page = async ({ params }: PageProps) => {
...
 
+    console.log(post._raw.sourceFileName.slice(0, -3))
```
Comment (jordan-bravo) [inline app/posts/[slug]/page.tsx:30]: Console log wanted?

File: contentlayer.config.js [hunk 4]
```diff
@@ -62,15 +62,15 @@ const computedFields = {
...
-        [
-          rehypeAutolinkHeadings,
-          {
-            properties: {
-              className: ["subheading-anchor"],
-              ariaLabel: "Link to section",
-            },
-          },
-        ],
+        // [
```
Comment (jordan-bravo) [inline contentlayer.config.js:65]: Commented code wanted?

**Review by jordan-bravo — APPROVED** [review]
I marked a couple of spots with console.logs and some commented code in case you wanted to get rid of them.

Otherwise looks good.

File: contentlayer.config.js (hunk 4 above)
Comment (sbddesign) [reply contentlayer.config.js:65]: This I want to keep, I may want to revisit this soon after the hackathon

File: components/MDX-components.tsx (hunk 1 above)
Comment (sbddesign) [reply components/MDX-components.tsx:92]: Fixed

File: app/posts/[slug]/page.tsx (hunk 2 above)
Comment (sbddesign) [reply app/posts/[slug]/page.tsx]: Fixed

**Pull request merged**